scripts/simex

```diff
@@ -824,6 +824,25 @@ def do_queue_kill(args):
 queue_kill_parser = queue_subcmds.add_parser('kill')
 queue_kill_parser.set_defaults(cmd=do_queue_kill)
 
+def do_queue_show(args):
+	queue_info = util.yaml_from_string(simexpal.queuesock.show_queue().decode())
```

Comment:
Decoding and parsing of YAML should be done way up in the stack, probably in sendrecv.